source/versioned-api/tests/crud-api-version-1-strict.yml

```diff
@@ -61,7 +61,28 @@ tests:
                 pipeline: *pipeline
                 <<: *expectedApiVersion
 
-  - description: "aggregate on database appends declared API version"
+  - description: "aggregate on database appends declared API version on servers before 4.9.0"
```

Comment:
I don't understand why we need this test.  When would any driver be executing it, given that we all use "latest" (which I think is 4.9)?

Reply:
This came up when @p-mongo started testing on 4.7.0. Not sure if we explicitly need the test, I'm personally fine restricting the other test to 4.9 and newer.

Reply:
That's what i would prefer.  We shouldn't need to specify behavior for internal releases where the behavior changes in the next internal release.

Reply:
I removed the extra test, so the only change that remains is restricting the existing test to 4.9.0 and above.